### ISSUE TITLE
Chrome 73. Manual control does not work in iframe

### ISSUE THREAD

**Issue by kpa9021** [pupunzi/jquery.mb.YTPlayer#433]
it works fine in previous chrome. But since 14 march (73v release), manual control does not working as expecting.
Example: https://jsfiddle.net/pavel_k_01/0uhqeryc/ 
When i press "play", nothing happens. If i press "mute", then "play", then "unmute", i see a message in the log: "Unmuting failed and the element was paused instead because the user didn't interact with the document before."

**Comment by pupunzi** [comment]
Hi,
I can't replicate your issue.
Does this demo page work for you?
https://pupunzi.com/mb.components/mb.YTPlayer/demo/demo.html

The latest browser policy about the autoplay of media doesn't allow any media play automatically on page load if the audio is turned on.
https://developers.google.com/web/updates/2017/09/autoplay-policy-changes

If the audio is turned on, the YTPlayer plugin makes the video start playing after the first user interaction on the page; if it is turned off the video will start immediately.

Let me know,
Matteo

**Comment by kpa9021** [comment]
Hello. Yes, demo works fine.
But in chrome 73, "unmute" button stopped working in the iframe.
When I press "mute", then "play", then "unmute", the video is paused and a message appears in the log as in the screenshot: https://image.prntscr.com/image/nOsjFBhpQ76JYBw70-qvOg.png 
But it is strange, because I was interacting with the documen. I pressed the buttons 3 times.
I tested on win7 and win10

**Comment by pupunzi** [comment]
On your jsFiddle example page you forget to load the CSS. I suggest you test it in a real environment to be sure it doesn't work. As you can see the demo page is working and all my installations are working fine without performing your issue with mute/unmute action. 

![ytp](https://user-images.githubusercontent.com/56659/55342098-4592d380-54a8-11e9-820b-0aab4db69cfc.png)

**Comment by kpa9021** [comment]
Yes, the demo works fine. And the problem is not in styles. The problem happens when player is used inside iframe.
When i using pure js + youtube player, i have same issue. Example: https://jsfiddle.net/pavel_k_01/baqc6ydg/1/ . 
So, promlem is not in jquery.mb.YTPlayer
